src/main/jbake/content/blog/2017/03/24/distributed-tracing-with-camel.adoc

```diff
@@ -0,0 +1,267 @@
+= Distributed Tracing with Apache Camel and OpenTracing
+Gary Brown
+2017-03-24
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, apache, camel
+
+In this post we will discuss how the http://opentracing.io/[OpenTracing standard] is being supported in the upcoming release of Apache Camel (version 2.19).
+
+OpenTracing is the vendor neutral open standard for distributed tracing, supported across many languages and with a growing number of tracer implementations and framework integrations.
+
+ifndef::env-github[]
+image::/img/blog/2017/2017-03-24-camel-loanbroker-jms-instance.png[caption="Figure 1: ", title="Distributed trace for the Camel Loan Broker JMS example"]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/blog/2017/2017-03-24-camel-loanbroker-jms-instance.png[caption="Figure 1: ", title="Distributed trace for the Camel Loan Broker JMS example"]
+endif::[]
+
+=== How to Instrument a Camel Application
```

Comment:
I think you should out-dent all those headings one level (title = h1 - here you are at h3)